app/utils/listEmptyChecker.sc

```diff
@@ -0,0 +1,25 @@
+import akka.util.ByteString
```

Comment:
Lingering scala worksheet, I assume this should be deleted.

Reply:
whoops 😶